cfgov/jinja2/v1/_includes/molecules/pagination.html

```diff
@@ -1,63 +1,79 @@
 
 {# ==========================================================================
 
-   Pagination
+   pagination.render()
 
    ==========================================================================
 
    Description:
 
    Builds pagination markup when given:
 
-   posts:                        TODO: fill in data type and description.
+   total_pages: Total number of pages.
 
-   posts.paginator:              TODO: fill in data type and description.
+   curr_page:   Currently selected page (out of the total).
```

Comment:
I generally don't like abbreviations like this because it doesn't give you much and leads to confusion.